regolith/helpers/a_expensehelper.py

```diff
@@ -84,16 +87,28 @@ def construct_global_ctx(self):
         gtx["zip"] = zip
 
     def db_updater(self):
+        gtx = self.gtx
+
         rc = self.rc
-        if not rc.date:
-            now = dt.date.today()
+
+        if not rc.begin_date:
+            begin_date = dt.date.today() # string with date time format
+            begin_date = begin_date.strftime('%Y-%m-%d')
         else:
-            now = date_parser.parse(rc.date).date()
-        if not rc.due_date:
-            due_date = now + relativedelta(years=1)
+            begin_date = rc.begin_date() # need to raise exception for wrongly formatted user input
+        if not rc.end_date:
+            end_date = dt.date.today()
+            end_date = end_date.strftime('%Y-%m-%d')
```

Comment:
I think it is better to leave it as a datetime object.  yml can save datetime and date is in the schema as an allowed type

Reply:
done

Reply:
your comment says done but I don't see it done?  Something not pushed?